.circleci/config.yml

```diff
@@ -156,7 +156,11 @@ jobs:
           key: lint-{{ arch }}-{{ .Branch }}-{{ checksum "package-lock.json" }}
           paths:
             - "node_modules"
-
+  performance-tests:
+    executor: check-image
+    steps:
+      - checkout
+      - run: npm run test:e2e:performance || true
```

Comment:
the true is to not fail the pipeline (at least in this initial iteration) when performance tests fail